core/templates/services/site-analytics.service.ts

```diff
@@ -65,9 +65,9 @@ export class SiteAnalyticsService {
   registerNewSignupEvent(): void {
     this._sendEventToGoogleAnalytics('SignupButton', 'click', '');
   }
-  registerClickBrowseLibraryButtonEvent(): void {
+  registerClickBrowseLessonsButtonEvent(): void {
     this._sendEventToGoogleAnalytics(
-      'BrowseLibraryButton', 'click',
```

Comment:
Should we keep a record of the removed key on the top of this file so that we can use it for reference when reading the analytics data? (not sure!)

Reply:
The version control history should be sufficient IMO.